app/src/ingestion/scrapy_dst/spiders/ca_public_charge_spider.py

```diff
@@ -41,22 +37,30 @@ class CaPublicChargeSpider(CrawlSpider):
         ),
     )
 
-    def parse_page(self, response: HtmlResponse) -> dict[str, str | AccordionSections]:
+    def parse_page(self, response: HtmlResponse) -> dict[str, str]:
         self.logger.info("Parsing %s", response.url)
         extractions = {"url": response.url}
         title = response.css("title::text").get().removesuffix("| Keep Your Benefits")
         extractions["title"] = title.strip()
-        base_url = response.url
 
         # remove icon text
         response.css("div.ic-icon").drop()
 
-        # extractions |= self.parse_module_full(base_url, response.css("div.module-full"))
-        extractions |= self.parse_main_content(base_url, response.css("div.GTM-1"))
-
+        self.resolveAnchorHrefs(response)
+        extractions |= self.parse_main_content(response.css("div.GTM-1"))
```

Comment:
Consider using .update() for better readability instead of |=
```suggestion
        extractions.update(self.parse_main_content(response.css("div.GTM-1")))
```

Reply:
We use `|=` throughout our code. Plus using `.update()` adds another pair of parentheses which decreases readability. 🤷  I'll leave as it for consistency with other code.